docs/source/fitting/fitfunctions/FickDiffusion.rst

```diff
@@ -13,7 +13,11 @@ Models the Q dependence of the QENS line width (Gamma (hwhm)), diffusion coeffic
 residence times (tau) and jump lengths (l) to extract the associated long range diffusive
 motions of molecules. Fick's law for diffusion has the form:
 
-.. math:: Gamma(Q) = D*Q^2 where D=<l^2>*Q^2/6*tau
+.. math::
+
+   Gamma(Q) = D \cdot Q^{2},
+
+where :math:`D=<l^{2}> \cdot \frac{Q^2}{6} \cdot tau`
```

Comment:
I think the spacing with <> doesn't quite look right, maybe it should be 
```suggestion
where :math:`D=\langle{l^2}\rangle \cdot \frac{Q^2}{6} \cdot tau`
```
or prepend \left and \right onto the symbols.